src/main/java/de/tum/in/www1/artemis/config/StaticCodeAnalysisConfigurer.java

```diff
@@ -30,8 +30,9 @@ public StaticCodeAnalysisConfigurer() {
 
     @PostConstruct
     private void init() {
-        languageToDefaultCategories = Map.of(ProgrammingLanguage.JAVA, createDefaultCategoriesForJava(), ProgrammingLanguage.SWIFT, createDefaultCategoriesForSwift());
-        log.debug("Initialized default static code analysis categories for JAVA");
+        languageToDefaultCategories = Map.of(ProgrammingLanguage.JAVA, createDefaultCategoriesForJava(), ProgrammingLanguage.SWIFT, createDefaultCategoriesForSwift(),
+                ProgrammingLanguage.C, createDefaultCategoriesForC());
+        log.debug("Initialized default static code analysis categories for JAVA, SWIFT and C");
```

Comment:
Couldn’t we generate the list of languages in code? So we don’t rely on this fixes debug log message getting outdated again.